queue/src/fiskaltrust.Middleware.Localization.QueueDE/Repositories/DailyClosingRepository.cs

```diff
@@ -76,8 +76,8 @@ public async Task<ftQueueItem> GetQueueItemOfMissingIdAsync(ftActionJournal acti
             var receiptNumerator = JsonConvert.DeserializeAnonymousType(actionJournal.DataJson, new { ftReceiptNumerator = 0L }).ftReceiptNumerator;
 
             // QueueItems are stored before the ActionJournals, so we need to look for the QueueItem with the closest timestamp before the respective ActionJournal
```

Comment:
This sentence says that we are looking for the QueueItem with the closest timestamp **_BEFORE_** the respective ActionJournal and then in lines before we are searching for queue items with the closest timestamp **_after_** the respective ActionJournal. Am I missing something or something sounds wrong here?

Reply:
The queueitem is updated later